### PR TITLE
py-netifaces: update to 0.10.9

### PR DESCRIPTION
**Pull request by someuser12**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6, 10.14.6
Xcode 9.1, 10.3

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-netifaces/Portfile [hunk 1]
```diff
@@ -14,18 +14,17 @@ description         an easy way to get the address(es) of the machine’s networ
 long_description    ${description}
 
 homepage            https://alastairs-place.net/projects/netifaces/
-master_sites        ${homepage}
-
 distname            netifaces-${version}
```
Comment (reneeotten) [inline python/py-netifaces/Portfile:17]: you should be able to remove the `distname` field as well, this is the default set by the `python` PG

File: python/py-netifaces/Portfile [hunk 2]
```diff
@@ -14,18 +14,17 @@ description         an easy way to get the address(es) of the machine’s networ
...
+                    size    28844
 
-python.versions     27
+python.versions     27 36 37 38
 
 if {${name} ne ${subport}} {
     depends_build-append port:py${python.version}-setuptools
+    livecheck.type      none
+} else {
+    livecheck.name  netifaces
```
Comment (reneeotten) [inline python/py-netifaces/Portfile:29]: this `else` block shouldn't be necessary; the default `pypi` livecheck set by the `python` PG should be sufficient

File: python/py-netifaces/Portfile (hunk 1 above)
Comment (someuser12) [reply python/py-netifaces/Portfile]: Done, thanks for the review.

File: python/py-netifaces/Portfile (hunk 2 above)
Comment (someuser12) [reply python/py-netifaces/Portfile]: Done, thanks for the review.

**Pull request merged**